### PR TITLE
OHSS-36254: Move ACS addon cluster scoped objects perms

### PR DESCRIPTION
**Pull request by kurlov**
### What type of PR is this?
Fix

### What this PR does / why we need it?
This fixes permission issue for ACS engs to see ACS addon CRs via backplane.
There are cluster scoped CRs located in the namespace scoped config  by mistake. The PR fixes the wrong configuration and moves cluster scoped ACS Addon permissions to cluster config. The following resources moved:
```
  - clusterpackages
  - clusterobjectdeployments
  - clusterobjectsets
  - clusterobjecttemplates
  ```

### Which Jira/Github issue(s) this PR fixes?

Fixes OHSS-36254

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [x] Included documentation changes with PR
~- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:~

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

**Comment by Nanyte25** [comment]
@iamkirkbater Can we get this approved ?

File: docs/backplane/requirements/acs/10-acs-managed-service.md [hunk 1]
```diff
@@ -136,17 +140,13 @@ Permissions at cluster scope.
...
+* view addons
+* view addoninstance
+* view addonoperators
 * view packages
 * view objectdeployments
 * view objectsets
 * view objecttemplates
-* view addons
-* view addoninstance
-* view addonoperators
```
Comment (jewzaam) [inline docs/backplane/requirements/acs/10-acs-managed-service.md:149]: Why is this removed from docs?  Premissions still exist at project scope.

**Review by jewzaam — CHANGES_REQUESTED** [review]

File: docs/backplane/requirements/acs/10-acs-managed-service.md (hunk 1 above)
Comment (kurlov) [reply docs/backplane/requirements/acs/10-acs-managed-service.md:149]: It's only moved. There are two lists: namespace and cluster scopes

**Comment by kurlov** [comment]
@jewzaam is it expected that PR is not merged even though it's approved? Could it be because of `change requested` status?

**Comment by jewzaam** [comment]
The lgtm was removed because of content changes.  Need to get another lgtm.
